test/utils.ts

```diff
@@ -29,7 +29,7 @@ export interface TestSuite {
  * If no test suite exists, we return this object with `isMissing: true`
  * so that we can report it.
  */
-interface LoadedTestSuite {
+export interface LoadedTestSuite {
```

Comment:
These tests can be in the normal format, instead of the special API format (you don't need the API to test them). See the model tests for examples on how to write them.

Reply:
If any tests are written to need the client, then rewrite the tests to not need it.